src/Centreon/Application/Normalizer/IconUrlNormalizer.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
```

Comment:
```suggestion
 * Copyright 2020 Centreon (https://www.centreon.com/)
```

Reply:
I let it like cause we set this interval in all other headers